spec/sonar/cli_spec.rb

```diff
@@ -82,6 +82,29 @@
         end
       end
     end
+
+    describe 'sonar types command' do
+      it 'returns all sonar search types' do
+        output = run_command('types')
+        expect(output).to match(/Certificate to IPs/)
+      end
+    end
+
+    describe 'search_all command' do
```

Comment:
Agreed, I'll see if I can do that easily.  That was my initial approach but it involved adding "all" as a search type itself.  I don't like that method but I have an idea for another way.